saltnanny/scripts.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
```

Comment:
not sure I'm a huge fan of the name of the file as not very descriptive - but I think all the code is fine.